packages/react-reconciler/src/__tests__/ReactSuspense-test.internal.js

```diff
@@ -1322,4 +1322,161 @@ describe('ReactSuspense', () => {
       jest.advanceTimersByTime(1000);
     });
   });
+
+  it('calls unstable_do_not_use_getDOMNodes once when component unsuspends', () => {
+    let children;
+    let counter = 0;
+    function Child() {
+      return 'Hi';
+    }
+    function App() {
+      return (
+        <Suspense
+          fallback={'Loading'}
+          unstable_do_not_use_getDOMNodes={suspenseChildren => {
+            children = suspenseChildren;
+            counter++;
+          }}>
+          <AsyncText text="A" ms={1000} />
+          <Child />
+          <Suspense fallback={'Loading'}>
+            <AsyncText text="B" ms={2000} />
+          </Suspense>
+          <Suspense fallback={'Loading'}>
+            <AsyncText text="C" ms={500} />
+          </Suspense>
+        </Suspense>
+      );
+    }
+    const root = ReactTestRenderer.create(<App />);
+    expect(children).toEqual(undefined);
+    jest.advanceTimersByTime(1000);
+    expect(Scheduler).toHaveYielded([
+      'Suspend! [A]',
+      'Suspend! [B]',
+      'Suspend! [C]',
+      'Promise resolved [C]',
+      'Promise resolved [A]',
+    ]);
+    expect(Scheduler).toFlushAndYield(['A', 'C']);
+    expect(root).toMatchRenderedOutput('AHiLoadingC');
+    expect(children).toEqual([
```

Comment:
Need some tests that render host nodes, not just text. Also you can use `toMatchRenderedOutput`. That will make it easier to convert these to a different renderer, like noop, which is the plan for all the tests in this file. In fact... do you mind moving this to `ReactSuspenseWithNoopRenderer-test.internal.js` :D